packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/ports.controller.js

```diff
@@ -12,11 +12,13 @@ export default /* @ngInject */ function XdslModemPortsCtrl(
   PackXdslModemPortObject,
   tucValidator,
   TucPackXdslModemMediator,
+  PACK_XDSL,
```

Comment:
You could directly import the constants, no need to have an `angular.constant` here